packages/config/config/devices/0x0346/keypad.json

```diff
@@ -0,0 +1,387 @@
+// Ring Keypad
+// Param information taken from https://support.ring.com/hc/en-us/article_attachments/360051851012/Keypad_Zwave_UK.pdf
+{
+	"manufacturer": "Ring",
+	"manufacturerId": "0x0346",
+	"label": "Keypad",
+	"description": "Keypad",
+	"devices": [
+		{
+			"productType": "0x0101",
+			"productId": "0x0201"
+		},
+		{
+			"productType": "0x0101",
+			"productId": "0x0202"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Battery Report Interval",
+			"unit": "min",
+			"valueSize": 2,
+			"minValue": 70,
+			"maxValue": 1440,
+			"defaultValue": 70,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Z-Wave Sleep Timeout",
+			"unit": "sec",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 5,
```

Comment:
Is it really seconds if the max is 5? Seems short but could be.

Reply:
No, that can be. The specs say <= 10 seconds for sleep timeouts

Reply:
Let me check what happens if I set it to more than 5. The PDF I linked to in the bottom of the config say: "Z-Wave Sleep Timeout: Time to wait after receiving ZW for more OTA  messages. Available settings: 1-5 seconds".

Reply:
Tried it. Setting it to 6 doesn't do anything. It just keeps the old value. But setting it any value between 1-5 works. So I would say that the PDF is correct.